clap_derive/tests/issues.rs

```diff
@@ -31,15 +31,15 @@ fn issue_151() {
 
 #[test]
 fn issue_289() {
-    use clap::{AppSettings, Clap};
-
     #[derive(Clap)]
     #[clap(setting = AppSettings::InferSubcommands)]
     enum Args {
         SomeCommand(SubSubCommand),
         AnotherCommand,
     }
 
+    // FIXME (@CreepySkeleton): current implementation requires us to
+    // derive IntoApp here while we don't really need it
```

Comment:
This comment is not needed anymore, is it?

Reply:
This is not really fixed, I've simply put a hack (the `is_subcommand` method) which I would like to get rig of. Related to my comment above.